### PR TITLE
Remove the NSString category and use the one provided by the content SDK

### PR DESCRIPTION
**Pull request by crousselle**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @crousselle has signed the CLA. Thanks for the pull request!

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m [hunk 1]
```diff
@@ -7,10 +7,11 @@
 //
 
 #import "UIImage+BOXBrowseSDKAdditions.h"
-#import <BoxContentSDK/BoxContentSDK.h>
-#import <BoxBrowseSDK/BoxBrowseSDK.h>
-#import "NSString+BOXBrowseSDKAdditions.h"
 #import "NSBundle+BOXBrowseSDKAdditions.h"
+#import <BoxBrowseSDK/BoxBrowseSDK.h>
+
+@import BoxContentSDK.BOXContentSDK;
```
Comment (h2kuo) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: Is this different than just ```@import BoxContentSDK;```?

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m [hunk 2]
```diff
@@ -7,10 +7,11 @@
...
+@import BoxContentSDK.NSString_BOXContentSDKAdditions;
```
Comment (h2kuo) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: Do we still need this line with the above import?

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m (hunk 1 above)
Comment (crousselle) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: Not sure. Theoretically, this should import whatever is imported in the BOXContentSDK.h file. The categories are not in this file. 
I wonder what the best practice is : @import BoxContentSDK or @import BoxContentSDK.BOXContentSDK.

I'll try to read on this subject, in the meantime, fixed it.

File: BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m (hunk 2 above)
Comment (crousselle) [inline BoxBrowseSDK/BoxBrowseSDK/Categories/UIImage+BOXBrowseSDKAdditions.m]: removed the previous line as it was not required;

**Comment by h2kuo** [comment]
+2

**Pull request merged**